<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrWindow</class>
 <widget class="QWidget" name="InstrWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>440</width>
     <height>440</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Don't have an account yet?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_instrSignupFromLogin">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(29, 161, 242);</string>
     </property>
     <property name="text">
      <string>Sign up!</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_InstrLogin">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>321</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Instructor Login</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>46</y>
        <width>263</width>
        <height>126</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Email"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Login!</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_instrSignup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Instructor Sign Up!</string>
     </property>
     <widget class="QPushButton" name="pushButton_instrSignup">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>BusyCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Sign Up!</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>48</y>
        <width>351</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Institution</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Institution"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Email_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Password_2">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Repeat Password</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_RepeatPassword">
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_modifyParams">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>361</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Modify Game Parameters</string>
     </property>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Holding Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BackOrder Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Weeks</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Inventory</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Shipment Delay</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Order Delay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifyParams">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_holdingCost">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_backOrderCost">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_startingInventory">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_totalWeeks">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>26</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_orderDelay">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_shipmentDelay">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>No.of Teams</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_noOfTeams">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QGroupBox" name="groupBox_modifyParams_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>371</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>Preview Game Parameters</string>
     </property>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Holding Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BackOrder Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Weeks</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Inventory</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Shipment Delay</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Order Delay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_startGame_instr">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Start Game!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_holdingCost_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_backOrderCost_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_startingInventory_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_totalWeeks_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>26</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_orderDelay_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_shipmentDelay_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>No.of Teams</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_noOfTeams_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Double-check your parameters before starting.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_preview_goBack">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Go Back!</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
